frontend/package-lock.json

```diff
@@ -6058,6 +6059,23 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/react-data-table-component": {
+      "version": "7.6.2",
+      "resolved": "https://registry.npmjs.org/react-data-table-component/-/react-data-table-component-7.6.2.tgz",
+      "integrity": "sha512-nHe7040fmtrJyQr/ieGrTfV0jBflYGK4sLokC6/AFOv3ThjmA9WzKz8Z8/2wMxzRqLU+Rn0CVFg+8+frKLepWQ==",
+      "dependencies": {
+        "deepmerge": "^4.3.1"
+      },
+      "peerDependencies": {
+        "react": ">= 16.8.3",
+        "styled-components": ">= 5.0.0"
+      },
+      "peerDependenciesMeta": {
+        "styled-components": {
+          "optional": false
+        }
+      }
+    },
     "node_modules/react-dom": {
```

Comment:
I just noticed this, but when doing `npm install` I get a diff on this file with:

```patch
+    "node_modules/react-data-table-component/node_modules/deepmerge": {
+      "version": "4.3.1",
+      "resolved": "https://registry.npmjs.org/deepmerge/-/deepmerge-4.3.1.tgz",
+      "integrity": "sha512-3sUqbMEc77XqpdNO7FRyRog+eW3ph+GYCbj+rK+uYyRMuwsVy0rMiVtPn+QJlKFvWP/1PYpapqYn0Me2knFn+A==",
+      "engines": {
+        "node": ">=0.10.0"
+      }
+    },
```

Reply:
I no longer get a diff, but looking at the PR, the changes are more than I'd expect from adding one dependency. Did you end up deleting the lock file and regenerating? If so, I think that may be why as it looks like a bunch of version updates. That's probably fine since ROPE is still in MVP development, though :man_shrugging: